createdb.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python
 
+from __future__ import print_function
```

Comment:
PEP 3105 has all the details: https://www.python.org/dev/peps/pep-3105/

In Python 2,  print is a statement, so print("foobar") means,
        -> pass a tuple () with one element "foobar" to the statement

In Python 3, print() is a function.  print("foobar") means pass one string the function.

The differences are not much, but they are there, and subtle.  I prefer just to use
**from __future__ import print_function**.   It was backported to Python 2.6.0a2: https://docs.python.org/2/library/__future__.html
and keeps the semantics of the print() function in Python 3, thus future proofing the code.